Dockerfile

```diff
@@ -67,18 +72,18 @@ RUN /tmp/scripts/holohub setup && rm -rf /var/lib/apt/lists/*
 # Enable autocomplete
 RUN echo ". /etc/bash_completion.d/holohub_autocomplete" >> /etc/bash.bashrc
```

Comment:
This is also necessary for anyone who wants to just run the application (without developing anything), right? Does it make sense to merge it with `holohub-cli-setup`?

Reply:
I see where you're coming from, but the goal of the stages here should be:
1. Set up any prerequisites for the HoloHub CLI
2. Use HoloHub CLI to set up the development container, i.e. prepare a common build and runtime environment

We should keep packages for general user and dev experience in this stage. Is there a better name than `dev` you would suggest?

Reply:
Maybe just `holohub-cli`? Not a blocker.

Reply:
Updated 👍